packages/api/schema/output.json

```diff
@@ -161,8 +244,7 @@
       },
       "required": [
         "calibrationDate",
```

Comment:
Should we remove this too, since `calibrationData` has a sequence of dates? Then models can choose the relevant date to calibrate against.